src/server/services/Actions.ts

```diff
@@ -45,24 +38,21 @@ export default class Actions {
       waitReady().then(() => {
         this.account = keyring.addFromMnemonic(mnemonic);
 
-        this.updateFaucetBalance().then(() =>
-          logger.info('Fetched faucet balance 💰')
-        );
         // We do want the following to just start and run
         // eslint-disable-next-line @typescript-eslint/no-floating-promises
         // TODO: Adding a subscription would be better but the server supports on http for now
-        setInterval(() => this.updateFaucetBalance(), balancePollIntervalMs);
+        const updateFaucetBalance = (log = false) => {
```

Comment:
why have `log = false` if the only usage is `updateFaucetBalance(true)`?

Reply:
To avoid polluting the logs with timeouted updates later, so it only logs first time on deploy and server run